charts/camunda-platform/README.md

```diff
@@ -908,6 +909,68 @@ elasticsearch:
   imageTag: <YOUR VERSION HERE>
 ```
 
+### Connectors
```

Comment:
Let's put it before the Elasticsearch section.

Reply:
Will do in the next revision.